### PR TITLE
Render Children Compatible with Hooks API

### PR DESCRIPTION
**Pull request by hew**
Hey everyone 👋🏽

This code allows you to render `React.element` using jsx3, from your existing ReForm code. Not sure if you have a need for this, but thought I'd share.

### CODE REVIEW

File: packages/bs-reform/re/ReForm.rei [hunk 1]
```diff
@@ -60,14 +60,15 @@ module Create:
...
     };
-    let component:
-      ReasonReact.componentSpec(
-        state,
-        ReasonReact.stateless,
-        ReasonReact.noRetainedProps,
-        ReasonReact.noRetainedProps,
-        action,
-      );
+    /* let component: */
```
Comment (fakenickels) [inline packages/bs-reform/re/ReForm.rei]: is this necessary?

**Comment by fakenickels** [comment]
Thank you @hew! Amazing
The code looks good, I just have one comment and I think we're good to go

File: packages/bs-reform/re/ReForm.rei (hunk 1 above)
Comment (medson10) [reply packages/bs-reform/re/ReForm.rei]: looks like it can be removed indeed

**Comment by hew** [comment]
Whoops. Forgot about that commented out type. I've deleted that now.

**Comment by fakenickels** [comment]
Released under v7.0.0, thanks!

**Comment by fakenickels** [comment]
@all-contributors please add @hew for maintenance

**Pull request merged**